inc/languages/english/admin/forum_attachments.lang.php

```diff
@@ -72,11 +72,11 @@
 $l['step1of2_line1'] = "Please wait, the file system is currently being scanned for orphaned attachments.";
 $l['step2of2_line1'] = "Please wait, the database is currently being scanned for orphaned attachments.";
 $l['step_line2'] = "You'll automatically be redirected to the next step once this process is complete.";
-$l['scanning'] = 'Scanning..';
+$l['scanning'] = 'Scanning…';
```

Comment:
Why its 2 here?

Reply:
Hard to tell. Maybe its some mistake, who left it there?

Reply:
I looked at it and it was added by this commit to language files:
https://github.com/mybb/mybb/commit/9dc1d0a73fae4e2ca0775925e245c500bbd01dfa#diff-bd2eb23b594576096e0256c0f4d8d62d ( https://github.com/mybb/mybb/issues/616) from this commit https://github.com/MyBBIr/mybb/commit/44de55595b5b1fc4014d9b80e6aa9b0017ae93bb

This was there for a long time.